pubsub/streaming-analytics/PubSubToGCS.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import datetime
+import json
+import logging
+
+import apache_beam as beam
+import apache_beam.transforms.window as window
+from apache_beam.options.pipeline_options import PipelineOptions
+
+
+class GroupWindowsIntoBatches(beam.PTransform):
+    """A composite transform that enriches Pub/Sub messages by publish
```

Comment:
Unless enriches is a Dataflow-specific term, I would just say, "A composite transform that groups Pub/Sub messages into windows based on timestamps added to the messages."